src/_main_/settings.py

```diff
@@ -287,6 +287,7 @@
 POSTMARK_DOWNLOAD_SERVER_TOKEN = os.environ.get('POSTMARK_DOWNLOAD_SERVER_TOKEN')
 SLACK_COMMUNITY_ADMINS_WEBHOOK_URL = os.environ.get('SLACK_COMMUNITY_ADMINS_WEBHOOK_URL')
 SLACK_SUPER_ADMINS_WEBHOOK_URL = os.environ.get('SLACK_SUPER_ADMINS_WEBHOOK_URL')
+SLACK_CELERY_TASKS_WEBHOOK_URL = os.environ.get('SLACK_CELERY_TASKS_WEBHOOK_URL')
```

Comment:
we have one superadmin notification channel.  Please don't use a different one for this.

Reply:
The super admin notifications channel is quite busy with various actions, events, testimonials, and other alerts, making it a bit challenging to quickly identify the cause of an alert at a glance. That's why I separated the Celery alerts into a different channel. Yesterday, @Opoku-Agyemang proposed some improvements to this feature, suggesting that we no longer need to use Slack as a logging platform for this purpose.

Reply:
There hasn't been a message in that channel in almost 2 months.  I prefer a single super admin alert Slack channel, which just important alerts that the super admin team need to take note of, not multiple channels.

Reply:
I will do that